src/ng/directive/input.js

```diff
@@ -1261,7 +1261,7 @@ function baseInputType(scope, element, attr, ctrl, $sniffer, $browser) {
       $browser.defer.cancel(timeout);
       timeout = null;
     }
-    if (composing) return;
+    if (composing && ev.originalEvent.data !== undefined) return;
```

Comment:
So, should `composing` be set to `false` if `composing $$ ev.originalEvent.data === undefined`?
Also, where does `originalEvent` come from?

Reply:
That's right. Because, In IE, 'compositionupdate' event still occurred although an user focused out of text input. and originalEvent comes from this text input.